src/core/Form/Checkbox/Checkbox.md

```diff
@@ -0,0 +1,40 @@
+```js
+import { Checkbox } from './Checkbox';
+<>
+  <Checkbox
+    id="1"
+    defaultChecked
+    hintText="This is an example hint text"
+    status="checked"
+  >
+    This is a regular checkbox that is checked and has a hint text
+  </Checkbox>
+
+  <Checkbox
+    hintText="This is an example hint text"
+    status="error"
+    statusText="You need to accept the terms and conditions to continue"
+  >
+    This is a regular checkbox with a hint text and an error message
+  </Checkbox>
+
+  <Checkbox defaultChecked disabled>
+    This is a disabled checkbox
+  </Checkbox>
+
+  <Checkbox />
+
+  <Checkbox id="2" variant="large">
```

Comment:
Do we support `<Checkbox.large>Large</Checkbox.large>` syntax? If so, should we also document it?

Reply:
Seems to be supported, so I'd add an example showing this as well.